pkg/extensions/search/search_test.go

```diff
@@ -1364,7 +1364,7 @@ func TestExpandedRepoInfo(t *testing.T) {
 		}
 		So(found, ShouldEqual, true)
 
-		err = signature.SignImageUsingCosign("zot-cve-test:0.0.1", port)
+		err = signature.SignImageUsingCosign("zot-cve-test:0.0.1", port, false)
```

Comment:
Should we update CheckManifestSignature code to include new signatures (oci_layout.go)?

Reply:
fixed